### DISCUSSION TITLE
犹在昨日 - 晦涩弗里曼的博客

### DISCUSSION THREAD

**Discussion by giscus** [obscurefreeman/blog#7] [Giscus]
# 犹在昨日 - 晦涩弗里曼的博客

怀念和好友联机的日子。

https://obscurefreeman.github.io/blog/%E6%97%A5%E8%AE%B0/%E7%8A%B9%E5%9C%A8%E6%98%A8%E6%97%A5/

**Comment by obscurefreeman** [comment]
之前测试评论时giscus突然不能用了，所以临时改成了utterances，我发现两个app各有所长，giscus和我网站的风格适配得很好，而utterances虽然没有中文，但是有一些giscus没有的组件。
